examples/exec_tty.py

```diff
@@ -0,0 +1,267 @@
+# This example used code(the `Interceptor` related part) from
+# http://sqizit.bartletts.id.au/2011/02/14/pseudo-terminals-in-python/
+# which is licensed under the MIT license:
+# Copyright (c) 2011 Joshua D. Bartlett
```

Comment:
Sorry that I couldn't get to this earlier. I'm concerned with the MIT licensing and worried if having another license violates our [CLA](https://github.com/cncf/cla/). Did a quick search in our code base and I didn't see any source code under MIT license in client-python and kubernetes/kubernetes (third party vendors don't apply). 

@yliaog @mbohlool Do we have existing best-practice / SIG in community that we can consult with?

I'd be happy to review & merge this PR after we sort out the license question.

Reply:
https://www.cncf.io/blog/2017/02/01/cncf-recommends-aslv2/

CNCF recommends Apache License. Not sure if MIT License is ok though.